server/RdtClient.Data/Enums/Provider.cs

```diff
@@ -13,6 +13,9 @@ public enum Provider
     [Description("Premiumize")]
     Premiumize,
 
+    [Description("DebridLinkFr")]
+    DebridLinkFr,
+
     [Description("TorBox")]
     TorBox
```

Comment:
```suggestion
    [Description("TorBox")]
    TorBox

    [Description("DebridLinkFr")]
    DebridLinkFr,
```

This way TorBox will keep `3`, and  `DebridLinkFr` will be `4`.

Reply:
done in https://github.com/rogerfar/rdt-client/pull/680/commits/d6d1813340115364e30b0f7e6b866abb277590bd